packages/terra-menu/src/MenuItem.jsx

```diff
@@ -275,6 +296,31 @@ class MenuItem extends React.Component {
     ]);
 
     let content = textContainer;
+
+    const screenReaderResponse = (
+      <>
+        { MenuUtils.isMac() && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.index' }, { index: index + 1, totalItems })} /> }
```

Comment:
Can share what happens if we include this for other browsers !! if this an issue only with safari can this be fixed with aria helpers or by modifying menu construct to make it more consistent across platform ??  ( Having platform specific instructions might behave different over the other versions of MacOS as the change in behaviour from version to version is expected. if this is an issue with Safari probably we should get the issue fixed on safari which would be more consistent for long run. )

Reply:
This issue is occurring for both chrome and safari. Even with the use of aria attributes, it is unable to match the response as seen in JAWS.

`aria-checked` works but gets announced only when navigated to the item, no context is provided when toggled by staying on the same item
 Index and selection state (aria-selected) doesn't apply when role=menuitem/menuitemcheckbox/menuitemradio (valid roles). 


`aria-expanded` for nested menu context does not provide helpful info to the user (both in VO and JAWS) and fails to announce expanded state due to shift in focus to submneu. 

Due to this `isMac` check is introduced for any browser in macOS